### DISCUSSION TITLE
Kogito Drools compatibility

### DISCUSSION THREAD

**Discussion by Emiliopg91** [quarkusio/quarkus#35393] [Community]
Good morning community

For my project I've been using Quarkus 2.X versions during early phases but now my company needs to upgrade to Quarkus 3.2.3.Final.
The problem is that during early phases, we integrated Kogito/Drools plugin and it worked in Quarkus 2.X but now, we migrated to Quarkus 3.2.3.Final and it seems not to have support for Kogito (no plugin version available, examples removed from quickstart...)

Has this plugin/module been deprecated? Will it be restored?

Can someone help me? We need to run DRL files and now that capacity is not available... any alternative?

**Comment by Abdullah-Soliman** [comment]
We are having the same problem, can someone from the Quarkus team opine on this.

**Comment by Sbruk** [comment]
Hello,

I'd be interested in this topic as well as I cannot start my application using Java 17 with the extension kogito-quarkus-rules as it complains about not finding the package javax.enterprise.context. I've found [this ticket](https://issues.redhat.com/browse/KOGITO-7907) on Kogito side but no updates there either.

Thanks for your help!
